index.html

```diff
@@ -0,0 +1,260 @@
+              </p>
+            </div>
+            <div class="d-30">
+              <ul id="tech-list" class="project-button-group">
+                <li>
+                  <span>
+                    HTML
+                  </span>
+                </li>
+              </ul>
+              <div class="d-flex">
+                <a id="btn-see-live" href="#" class="see-live project-button">
+                  See Live
+                  <img src="assets/images/icons/Icon - Export.png" alt="See Live Icon">
+                </a>
+                <a id="btn-see-source" href="#" class="see-source project-button">
+                  See Source
+                  <img src="assets/images/icons/Icon -GitHub.png" alt="See Source Icon">
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="welcome">
+      <!-- <img class="desktop" src="assets/images/hero_img.png" alt="" srcset="" />
+      <img class="mobile w-100" src="assets/images/hero_img_mobile.png" alt="" srcset="" /> -->
+      <h1 class="shake">Online Web Tutorials <br>Learn to Code <br>in 2023</h1>
+      <div>
+        <p class="brief">
+          Get certified by completing a course with the world's largest web developer site
+          optimized for learning and training with simplified examples to improve
+          reading and learning. Tutorials, references, and examples are constantly reviewed to 
+          avoid errors, but we cannot warrant full correctness of all content.
+        </p>
+      </div>
+      <h4 class="event-time">
+        2023.10.15(THU) ~ 16(FRI)
+      </h4>
+      <p class="event-venue">@ National Library Lagos, IT center Lekki and more.</p>
+    </section>
+    <section id="program">
+      <h2>Main Program</h2>
+      <hr class="mx-3"/>
+      <div class="program-grid">
+        <div>
+          <img src="assets/images/cp/program_icon_01.png" alt="" srcset="">
+          <h5>Lecture</h5>
+          <p>Listen to the speakers from various countries share their thoughts on industry standard practices.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_02.png" alt="" srcset="">
+          <h5>CC Exhibition</h5>
+          <p>Get a chance to get your various creations/designs  displayed in our website for potential employees.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_03.png" alt="" srcset="">
+          <h5>Forum</h5>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_04.png" alt="" srcset="">
+          <h5>Workshop</h5>
+          <p>Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <div>
+          <img src="assets/images/cp/program_icon_05.png" alt="" srcset="">
+          <h5>CC Ignite</h5>
```

Comment:
Kindly make sure that all your images contain alt attributes.
Image alt attributes are an essential part of web accessibility. They provide alternative text descriptions for images, allowing visually impaired users :kissing_closed_eyes:  or those using assistive technologies :studio_microphone:  to understand the content and purpose of the images.
Remember, alt attributes not only improve accessibility but also benefit search engine optimization (SEO) by providing relevant text descriptions :book:  that can be indexed by search engines. By adding alt attributes to your images, you contribute to a more inclusive and user-friendly web experience. :wink: